lona/html/node_list.py

```diff
@@ -65,7 +65,7 @@ def append(self, value):
 
             self._nodes.append(node)
 
-            index = self._nodes.index(node)
+            index = len(self._nodes) - 1
```

Comment:
What if you append to an empty node, where the length is `0`?  The resulting index would be `-1` in this case.

Reply:
I can't find a scenario that would result in the node list still being empty at that point. Even if the node is empty initially, `len()` is called after appending, so there should always be at least the one node in the list. 

An `append(None)` could maybe get us there, but that's caught by `check_value()` and results in a `ValueError`.

Reply:
I am sorry, you are completely right. I overlooked that the `len()` always comes behind the append to `self._nodes`. So, a negative number is not possible.